src/functions/asyncCredential/asyncCredentialHandler.ts

```diff
@@ -174,13 +174,25 @@ export async function lambdaHandler(
     return sessionRecoveredResponse(parsedRequestBody.sub);
   }
 
-  return {
-    headers: { "Content-Type": "application/json" },
-    statusCode: 200,
-    body: JSON.stringify({
-      message: "Hello World",
-    }),
+  const sessionConfig = {
+    authSessionId: "",
+    state: "",
+    sub: "",
+    client_id: "",
+    govuk_signin_journey_id: "",
+    redirect_uri: "",
+    issuer: "",
+    sessionState: "",
   };
+
+  const sessionServiceCreateSessionResult =
+    await sessionService.createSession(sessionConfig);
+
+  if (sessionServiceCreateSessionResult.isError) {
+    return serverError500Responses;
```

Comment:
please may we rename this to serverError500Response

Reply:
Done